iFixit/toolbox
# Toolbox: Document constants

Summary
---

Documentation for Toolbox style constants used to be very hard to find. It used to live here: https://constants-demo.netlify.com. This pull moves that documentation to https://ifixit.design so it's more accessible.

This pull just adds a usage example and reference table for each constant. In future pulls, we can add visual representations of the constant values.

![image](https://user-images.githubusercontent.com/4608155/46555255-8f06bb00-c897-11e8-9798-7c694938fee1.png)


QA
---

- Go to https://deploy-preview-50--toolbox.netlify.com/
- Assert that "Constants" is in the navigation menu
- Assert that all the navigation links work as expected

Closes https://github.com/iFixit/ifixit/issues/25995
